addons/escoria-core/game/core-scripts/esc/commands/camera_shift.gd

```diff
@@ -21,6 +21,9 @@
 extends ESCBaseCommand
 class_name CameraShiftCommand
 
+# The list of supported transitions as per the link mentioned above
+var supported_transitions = ["LINEAR","SINE","QUINT","QUART","QUAD" ,"EXPO","ELASTIC","CUBIC",
```

Comment:
Please make this a const and also CONSTANT_CASE the name. Thanks!

Reply:
Done @BHSDuncan